addons/redirect-mobile-forums-v2/addon.json

```
@@ -1,6 +1,6 @@
 {
   "name": "Redirect to standard forums",
-  "description": "Redirects you to the standard (\"main\") forums when you click a link to the mobile forums.",
+  "description": "Redirects you to the standard (\"main\") forums when you click a link to the mobile forums, avoiding Page not found errors.",
```

A description suggestion I proposed on #6917:

```suggestion
  "description": "Redirects you to the standard (\"main\") view of the forums when you follow a link to the mobile view of the forums (which doesn't work anymore)",
```

---

It should be clear that you get an error if you don't enable this addon.

---

Like this?
```suggestion
  "description": "Redirects you to the standard (\"main\") view of the forums when you follow a link to the mobile view of the forums, avoiding Page not found errors.",
```

---

> I think it should be visit instead of follow

"Visit" could work but may as well be changed from "visit a link to" to just "visit". Other alternative verbs include "open" and "go to".